packages/react-devtools-shared/src/devtools/views/Components/InspectedElementSuspenseToggle.js

```diff
@@ -53,7 +56,7 @@ export default function InspectedElementSuspenseToggle({
       </div>
       <div className={styles.ToggleSuspenseRow}>
         <span className={styles.Name}>Suspended</span>
-        {canToggleSuspense ? (
+        {!readOnly && canToggleSuspense ? (
```

Comment:
Can we move the new `readOnly` conditional to make it part of the `canToggleSuspense` option?

Also, can you explain the rationale behind checking `readOnly` here when we also have a `hideToggleSuspenseAction` check?

Reply:
Done.

> Also, can you explain the rationale behind checking `readOnly` here when we also have a `hideToggleSuspenseAction` check?

I wanted to support all combinations of the new flags, so `readOnly` could be `true` and `hideToggleSuspenseAction` `false`. I don't know if this combination actually makes sense though.